### ISSUE TITLE
Error I keep getting

### ISSUE THREAD

**Issue by ghost** [Mirreski/RandomSK#20]
http://pastebin.com/r9PYP56G

I think it has to do with my MC Spigot version, but I have no clue.

**Comment by Mirreski** [comment]
Yeah, some guy poped up in the irc and had a similar problem(He had some 1.8.3 snapshot he said). Seems it got fixed by updating spigot version.

It's actually a very strange error. ".....TinyProtocol$PacketInterceptor cannot be cast to .....TinyProtocol$PacketInterceptor". It cannot be cast into itself eh lol.

I could try see if there's a updated Tinyprotocol later. Adding it to my todoo list. (Will probably do this within a week). But you should try switching versions, like if you are using a snapshot try a stable build. Since well I test with 3 different versions and never got this error. Could you send me the jar which errors so I can have it as reference when I try after updating Tinyprotocol and see if it makes a difference for these erroring versions.

Tinyprotocol is used by On Vehicle Steer, On Sound, Sleep Effect, On Resource Pack Status and perhaps some more thingy.
